modelina-website/README.md

```diff
@@ -16,4 +16,32 @@ Here is a quick overview of where some of the functions for rendering the playgr
 - `src/components/playground/PlaygroundOptions.tsx` is the main component that renders the options based on which output is selected.
 - `src/components/playground/options` contain all the individual react components for showing the output options.
 - `src/components/playground/Playground.tsx` is the main playground component, and is the one rendered by the playground page.
-- `src/components/playground/GeneratedModels.tsx` is the playground component responsible for rendering the generated models.
\ No newline at end of file
+- `src/components/playground/GeneratedModels.tsx` is the playground component responsible for rendering the generated models.
+
+## Run Modelina Website locally
+
+Checkout documentation that explains [git-workflow](https://github.com/asyncapi/community/blob/master/git-workflow.md) used in our repositories.
+
+
+
+1. Navigate to the website directory.
+
+```bash
+    cd website-website
```

Comment:
@jonaslagoni @rukundob451 there is a still an error here 😓 
cd modelina-website

Reply:
I have checked this out now and have resolved it, thank you for the clarification.